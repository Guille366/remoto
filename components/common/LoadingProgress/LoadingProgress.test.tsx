import { render } from "@testing-library/react";
import Progress from "."

jest.mock("@tanem/react-nprogress", () => ({
    useNProgress: jest.fn(() => ({
        animationDuration: 200,
        isFinished: true,
        progress: 0
    }))
}));

describe('Loading', () => {
  it('should render properly', () => {
    const { container } = render(<Progress isAnimating />);

    expect(container.firstChild).toMatchSnapshot();
  });
});