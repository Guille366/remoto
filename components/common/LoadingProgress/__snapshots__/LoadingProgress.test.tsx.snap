// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render properly 1`] = `
<div
  class="pointer-events-none"
  style="opacity: 0; transition: opacity 200ms linear;"
>
  <div
    class="bg-violet-700 h-1 w-full top-0 left-0 fixed z-50"
    style="margin-left: -100%; transition: margin-left 200ms linear;"
  />
</div>
`;
